Test/PerformanceCollector/FunctionalTests/TestApps/TestApp40/Global.asax.cs

```diff
@@ -3,25 +3,63 @@
     using System;
     using System.Collections.Generic;
     using System.Configuration;
-    
+    using System.Diagnostics;
+    using System.IO;
+
     using Microsoft.ApplicationInsights;
     using Microsoft.ApplicationInsights.Extensibility;
     using Microsoft.ApplicationInsights.Extensibility.Implementation;
     using Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector;
+    using Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.QuickPulse;
     using Microsoft.VisualStudio.TestTools.UnitTesting;
 
     public class Global : System.Web.HttpApplication
     {
 
         protected void Application_Start(object sender, EventArgs e)
         {
+            // necessary for .NET CLR Memory counters to start reporting process ID
+            GC.Collect();
+
             var setting = ConfigurationManager.AppSettings["TestApp.SendTelemetyIntemOnAppStart"];
 
             if (false == string.IsNullOrWhiteSpace(setting) && true == bool.Parse(setting))
             {
                 new TelemetryClient().TrackTrace("Application_Start");
             }
 
+            PerformanceCollectorModule perfModule = InitializePerformanceCollectionModule();
+            
+            QuickPulseTelemetryModule quickPulseModule = InitializeQuickPulseModule();
+            
+            TelemetryModules.Instance.Modules.Add(perfModule);
+            TelemetryModules.Instance.Modules.Add(quickPulseModule);
+        }
+
+        private static QuickPulseTelemetryModule InitializeQuickPulseModule()
+        {
+            var quickPulseModule = new QuickPulseTelemetryModule();
+
+            quickPulseModule.QuickPulseServiceEndpoint = "http://localhost:7655/QuickPulseService.svc/";
```

Comment:
For safety in future code changes it would be good to concatenate the port 7655 in from a shared constant rather that typing it into multiple places.